src/main/kotlin/io/github/anki/anki/controller/CardsController.kt

```diff
@@ -64,42 +64,65 @@ class CardsController(
             required = false,
             defaultValue = PaginationDto.DEFAULT_OFFSET.toString(),
         ) offset: Int,
-    ): List<CardDtoResponse> =
-        cardService.findCardsByDeckWithPagination(
-            deckId = deckId,
-            userId = securityService.jwtUtils.getUserIdFromAuthHeader(header),
-            pagination = PaginationDto(limit, offset).toPagination(),
-        ).map { it.toDto() }
+    ): Mono<List<CardDtoResponse>> =
```

Comment:
```suggestion
    ): Flux<CardDtoResponse> =
```

Reply:
yes, i thought it will stream Cards to client